src/components/typography/typography.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { HTMLAttributes } from 'react';
+import styled from 'styled-components';
+import { up } from '../breakpoint/breakpoint';
+
+const H1 = styled.h1`
```

Comment:
I think we should stay semantic with the headlines. Clarify how to name H1/H2. It would be much more readable to if they are named like "PageTitle". "SectionTitle" etc. h1/h2 are kind of implementation details although we need to ensure a correct order for accessibility later of course.

Reply:
Good idea, done.